spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialConvolution.scala

```diff
@@ -368,8 +369,6 @@ class SpatialConvolution(
   }
 
   override def zeroGradParameters(): Unit = {
```

Comment:
maybe we can just remove the method. I remember the default impl is do nothing, right?

Reply:
No. The default implementation will get parameters and zero gradients.

```scala
  def zeroGradParameters(): Unit = {
    if (parameters() != null) {
      parameters()._1.zip(parameters()._2)foreach{ case (weight, grad) =>
        grad.resizeAs(weight).zero()
      }
    }
  }
```